### ISSUE TITLE
Report which WCAG success criteria relate to a11y rules

### ISSUE THREAD

**Issue by WilcoFiers** [ember-template-lint/ember-template-lint#2666]
After a conversation with @MelSumner, I've been looking to see how consistent ember-template-lint would be with [accessibility conformance test (ACT) rules](https://www.w3.org/WAI/standards-guidelines/act/rules/). In looking at that, one of the things I noticed was that the linter results include a information on what rule fails, but not what that rule might be related to.

There are lots of ways to do that. For example, the result could have an optional `wcag2` property which lists WCAG success criteria numbers. If you wanted to do something a little broader you could allow tags for example, and have WCAG success criteria numbers as a type of tag.

For developers it can be important to know if what is reported is a failure of WCAG, what level the failure is at, or even if it's not a WCAG a11y issue, but maybe a best practice.

**Comment by MelSumner** [comment]
I'd be willing to discuss this work and what it would need to make this idea happen @bmish @rwjblue 

One thing that something like this would help with is consistent (integration? collaboration?) with the ACT rule testing. 

Also noting for context that we've been having some discussion about exploring the inclusion of static analysis in the ACT rule space.

**Comment by bmish** [comment]
One consideration is that we have been trying to move toward matching ESLint's behavior (and perhaps even converting this to an ESLint plugin someday), so would be interesting to hear if there's precedent for anything like this in ESLint. If not, maybe there could be some other layer created that just maps template lint rules/violations to ACT rules.

**Comment by WilcoFiers** [comment]
I can't find anything in ESLint like this. The closest I've seen is categories, which is used to group rules, but that's probably not granular enough.
